docs/sdk/client-side-sdks/roku/roku-install.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Roku SDK Installation
+sidebar_label: Installation
+sidebar_position: 1
+description: Installing the SDK
+sidebar_custom_props: { icon: material-symbols:install-desktop }
+---
+
+[![GitHub](https://img.shields.io/github/stars/devcyclehq/roku-client-sdk.svg?style=social&label=Star&maxAge=2592000)](https://github.com/devcyclehq/roku-client-sdk)
+
+## Requirements
+
+This version of the DevCycle Client SDK supports the following platforms:
+
+- **Roku OS 9.4+**
+
+## Installation
+
+[//]: # 'wizard-install-start'
```

Comment:
we don't have it for wizard install start yet right?

Reply:
nope, but these tags will be used to automatically pull the right content when its enabled :)